roles/ffh.mesh_wireguard/templates/ferm.conf.j2

```diff
@@ -0,0 +1,15 @@
+domain (ip6) {
+  table filter {
+    chain INPUT {
+      interface ({{ mesh_wireguard_iface }}-{{ domains | map(attribute='id') | join(' '+ mesh_wireguard_iface + '-') }}) proto udp dport {{ vxlan_port }} ACCEPT;
+    }
+  }
+}
+domain (ip ip6) {
+  table filter {
+    chain INPUT_SOURCE_IS_NOT_FFH_EXIT {
+      interface eth0 proto udp dport 51810:51823 ACCEPT;
```

Comment:
We will forget to update this when we add more domains. Please keep this dynamic.

Reply:
sure :/
https://github.com/freifunkh/ansible/blame/3ab6a4d8eab2d1dbe07b0c7e82ff47a5b382b3f1/roles/ffh.mesh_fastd/templates/ferm.conf.j2